vector/src/main/java/im/vector/activity/VectorBaseSearchActivity.java

```diff
@@ -66,11 +67,19 @@
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        setContentView(R.layout.activity_vector_unified_search);
```

Comment:
Ah, sorry! I didn't even notice that this was being set in the superclass (and I got really confused). I'll fix this in the next update :smile: